index.js

```diff
@@ -0,0 +1,37 @@
+var program = require('commander');
+var requireDir = require('require-dir');
+var package = require(__dirname + '/package.json');
+
+program.version(package.version)
+  .option('-D, --debug', 'enable debugging and debug output')
+  .option('-q, --quiet', 'quiet output, except for errors')
+  .option('-v, --verbose', 'verbose output')
+  .option('-j, --json', 'JSON data output')
+  .option('--nocolors', 'no ANSI colors in JSON output');
+
+// Common command init wrapper with error reporting.
+function init (cmd) {
+  return function () {
+    var args = Array.prototype.slice.call(arguments, 0);
```

Comment:
Well, the notion here is that some code after the `var args` can filter & alter the args if necessary. Not doing that yet, but it was a thing I found handy in another project. But, might be worth leaving out since it hasn't proven interesting here so far.